src/main/kotlin/com/autonomousapps/tasks/SynthesizeDependenciesTask.kt

```diff
@@ -180,10 +183,6 @@ abstract class SynthesizeDependenciesTask @Inject constructor(
         .forEach { dependency ->
           val coordinates = dependency.coordinates
           outputDir.file(coordinates.toFileName()).get().asFile.bufferWriteJson(dependency)
-          if (coordinates is IncludedBuildCoordinates) {
-            // Write the same information using the 'project' coordinates as later processing steps rely on it
-            outputDir.file(coordinates.resolvedProject.toFileName()).get().asFile.bufferWriteJson(dependency)
```

Comment:
This was bound to cause issues – writing the same information twice with different file names. This is now no longer necessary as "inside" the analysis the `IncludedBuildCoordinates` notation is used consistently.